src/lottery-application.ts

```diff
@@ -4,21 +4,21 @@ import { TryAgainAlgorithm } from './algorithms/algorithm-v1';
 
 class Lottery {
 
-    private numbers: number[];
+    private readonly numbers: number[];
```

Comment:
Does _readonly_ works only by assigning a new object to this pointer or also blocks its internal properties?

Reply:
That is a mistake. I will remove that property.